tests/resources/joined_data.csv

```diff
@@ -0,0 +1,201 @@
+0,1916,4,17,0,8275 QI,97,I,1.0,147.82916666666668,483.92416666666674,1198.26875,24.914583333333336,0.0,0.0,16.916666666666668,2.25625,74.98333333333333,317.50708333333336,1.0,147.82916666666668,483.92416666666674,1198.26875,24.914583333333336,0.0,0.0,16.916666666666668,2.25625,74.98333333333333,317.50708333333336,1.0,128.19330275229356,206.87577981651376,1319.4381651376148,32.80091743119266,48.96733944954129,51.95678899082569,29.05119266055046,1.7339449541284404,21.88192660550459,113.28366972477065,1.0,116.49469879518072,229.4806024096385,986.734096385542,37.425,68.03716867469879,2.3836144578313254,59.46006024096386,1.8975903614457832,367.99385542168676,0.0,53.16897590361445,1.0,153.87704545454545,204.35742424242423,832.7903030303032,19.856590909090905,5.343712121212121,2.539772727272727,18.86795454545455,8.266439393939395,115.67007575757576,0.0,40.43833333333333,1.7223264540337708,1.0,161.18292682926833,294.04671669793623,1666.0221388367734,95.73377110694186,36.184803001876176,179.6831144465291,11.722889305816135,0.0,321.9578799249531,0.0,1.7223264540337708,114.83883677298313,12.622138836772985,1.0,182.9853114181036,393.76952035225713,1610.7689825552084,23.725622668145608,30.55339316325502,64.59019864878492,1.059527410843333,14.685355114113813,377.4149440354946,0.0,194.48216866659942,117.69190951564651,40.6296258949279,0,2013-01-07,72,79,13,73,40,68,5,79,26,85,79,27,28,30,9,25,29,65,43,72,70,35,77,77,93,43,37,48,80,61,21,5,92,93,14,28
+1,1904,12,3,0,6892 ZT,0,T,1.0,143.53043900072763,340.23162745573615,1127.425782197429,28.79153528983749,46.299660441426155,0.989328159107446,16.120058210041233,27.21937909289353,787.6514673781227,45.66092650982294,1.0,143.53043900072763,340.23162745573615,1127.425782197429,28.79153528983749,46.299660441426155,0.989328159107446,16.120058210041233,27.21937909289353,787.6514673781227,45.66092650982294,1.0,124.40923706518558,211.91493715287925,1393.5342005261618,13.65097924583455,7.263227126571177,13.382344343759133,6.7743349897690734,0.0,65.76483484361296,13.381759719380298,1.0,131.01382488479263,177.42140296979005,456.30547875064,55.194879672299024,0.0,33.43563748079877,11.054275473630312,11.97900665642601,140.31049667178698,0.0,107.70916538658471,1.0,159.54646153846156,215.888,699.692,16.253076923076925,11.737076923076925,37.203692307692315,4.9310769230769225,18.65246153846154,218.92461538461532,0.0,4.0958461538461535,0.0,1.0,114.70625,95.87,942.9320833333333,49.33333333333334,0.0,0.0,0.0,0.0,0.0,0.0,0.0,35.22416666666667,9.375,1.0,155.86588979303022,391.3936405448434,1239.6159782416416,49.39521050769503,18.585795153016093,73.68680346718557,6.174464885901291,18.76191845038033,299.71274102246593,0.0,3.3886874226074646,63.31571731823811,23.077812665841144,1,2012-07-25,66,57,70,92,23,39,66,75,16,90,37,86,62,58,54,76,89,27,40,48,75,35,42,42,31,52,78,38,24,15,33,25,47,95,8,96
```

Comment:
Not sure if you really need such a big complicated dataset to test the things that you want to test

Reply:
This is a part of the dataset that the master algorithm receives when running on `vantage6-local-setup`. I had a lot of issues with getting the pipeline to work on this while the iris dataset ran like a dream. But the dataset is a little large I admit, I'll remove most of the lines.